README.md

```diff
@@ -15,7 +15,7 @@ Jackson datatype module for [Vavr](http://vavr.io/) library
 <dependency>
   <groupId>io.vavr</groupId>
   <artifactId>vavr-jackson</artifactId>
-  <version>0.9.0</version>
+  <version>1.0.1</version>
```

Comment:
Well, sure. But shouldn't it match the version given in the POM in this project?